### DISCUSSION TITLE
Fetch is not defined at 'npx jsr publish' command

### DISCUSSION THREAD

**Discussion by danilockthar** [jsr-io/jsr#589] [Q&A] [answered]
Im receiving this error when i try to publish my package to JSR. What could be done ? I have tried to add node-fetch to the npx project in my local machine but i could not make it work.

```
const res = await fetch(DENO_CANARY_INFO_URL);
                ^

ReferenceError: fetch is not defined
```

**Comment by marvinhagemeister** [comment, marked as answer]
What node version are you on? You can check by running `node --version`. The `fetch`-API was added to node natively in v18.0.0

**Reply by danilockthar** [reply]
It was actually on version 16. Thank you!
